meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx8mmevk/lmp.cfg

```diff
@@ -1,39 +1,62 @@
+CONFIG_SPL_DM=y
 CONFIG_SPL_FIT=y
 CONFIG_SPL_FIT_SIGNATURE=y
 CONFIG_SPL_LOAD_FIT=y
+# CONFIG_SPL_FIT_IMAGE_TINY is not set
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_SPL_CRYPTO_SUPPORT=y
+CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_SPL_USB_HOST_SUPPORT=y
+CONFIG_SPL_USB_GADGET=y
+CONFIG_SPL_USB_SDP_SUPPORT=y
+CONFIG_RSA=y
+CONFIG_HASH=y
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
```

Comment:
Possiblty defining `CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR`  won't work, as it's redefined in `imx8mm_evk.h`: 
```
./include/configs/imx8mm_evk.h:#define CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR	(0x300 + CONFIG_SECONDARY_BOOT_SECTOR_OFFSET)
```

we can set `CONFIG_SECONDARY_BOOT_SECTOR_OFFSET` instead

Reply:
It's actually fine if this is already defined.  We can leave as is and remove from lmp.cfg.

Reply:
I will drop this line 14 in the next update of this PR.